src/main/scala/exercises/06_Collections.scala

```diff
@@ -130,7 +140,11 @@ object Collections {
    * neat way using recursion. 
    */
   def tails[T](l: List[T]): List[List[T]] = {
-    error("fix me")
```

Comment:
This is the same algorithm I used, but I think yours is more clear (I tried to pattern match, which I think I overused in this assignment).